content/docs/examples/multicluster/split-horizon-eds/index.md

```diff
@@ -60,9 +60,9 @@ This will be used to access pilot on `cluster1` securely using the ingress gatew
       --set global.controlPlaneSecurityEnabled=true \
       --set global.proxy.accessLogFile="/dev/stdout" \
       --set global.meshExpansion.enabled=true \
-      --set global.meshNetworks.network2.endpoints[0].fromRegistry=n2-k8s-config \
-      --set global.meshNetworks.network2.gateways[0].address=0.0.0.0 \
-      --set global.meshNetworks.network2.gateways[0].port=443 \
+      --set 'global.meshNetworks.network2.endpoints[0].fromRegistry=n2-k8s-config' \
```

Comment:
This is needed for z shell? Is it the `[]` chars? Is this not also a problem in many other docs?

Reply:
I found that some other docs didn't have this issue. The used \' or \"s so added that here.

A global search `[]` and [0]`, does highlight that other docs may have this issue. I'll add another commit fixing those up.

Reply:
Commit # 2 covers files with similar missing quotes around []'s.